python-sdk/nuscenes/scripts/export_instance_videos.py

```diff
@@ -255,21 +281,25 @@ def main(version, dataroot, output, object_categories, fps, output_size, minimum
         samples	-	Sensor data for keyframes.
         sweeps	-	Sensor data for intermediate frames.
         maps	-	Folder for all map files: rasterized .png images and vectorized .json files.
-        v1.0-*	-	JSON tables that include all the meta data and annotations. Each split (trainval, test, mini) is provided in a separate folder.
+        v1.0-*	-	JSON tables that include all the meta data and annotations.
+                    Each split (trainval, test, mini) is provided in a separate folder.
                     Note that image_annotations.json should be inside this directory.
     ```
     
     Args:
-        version (string): the NuScenes data version
+        version (string): the nuScenes data version
```

Comment:
Upper case of the first letters and full stop?

Reply:
nuScenes